packages/superset-ui-chart-composition/package.json

```diff
@@ -26,12 +26,12 @@
     "access": "public"
   },
   "dependencies": {
-    "@types/react": "^16.7.17",
+    "@types/react": "^16.9.32",
     "@vx/responsive": "^0.0.195",
     "csstype": "^2.6.4"
   },
   "peerDependencies": {
     "@superset-ui/core": "^0.12.0",
-    "react": "^16.7.17"
+    "react": "^16.13.1"
```

Comment:
I saw some typing errors related to `@types/react` and thought it'd be easier to just upgrade everything instead of hunting down which plugin imported the incompatible version. Will upgrade Superset main repo, too, when this is merged.